src/components/BottomNavbarWrapper/BottomNavbarWrapper.tsx

```diff
@@ -1,20 +1,16 @@
-import { useRouter } from "next/router";
 import BottomNavbar from "../BottomNavbar/BottomNavbar";
 import BottomNavbarWrapperStyled from "./BottomNavbarWrapperStyled";
-import userEndpoints from "@/utils/userEndpoints/userEndpoints";
+import { useAppSelector } from "@/store/hooks";
+import { useSession } from "next-auth/react";
 
 const BottomNavbarWrapper = (): JSX.Element => {
-  const route = useRouter();
-  if (
-    route.pathname === userEndpoints.login ||
-    route.pathname === userEndpoints.signup
-  ) {
-    return <></>;
-  }
+  const { isLogged } = useAppSelector((state) => state.user);
+  const { status } = useSession();
+  const isAuthenticated = status === "authenticated";
 
   return (
     <BottomNavbarWrapperStyled>
-      <BottomNavbar />
+      {(isLogged || isAuthenticated) && <BottomNavbar />}
```

Comment:
Em sembla bé el canvi. De totes maneres, crec té el model de render que hi havia abans en el sentit d'evitar que no es renderitzi el div de <BottomNavbarWrapperStyled />. És a dir, amb un if com hi havia abans, a sobre posar que si no està loggejat o authentificat que es renderitzi un fragment i així ens evitem que es renderitzi un div buit. No sé si m'explico!

Reply:
jaja no the entés massa aquí. Però així es renderitza un div buit?